### PR TITLE
tweaked landing page and made footer

### PR DESCRIPTION
**Pull request by theJayBecker**

### CODE REVIEW

File: src/app/index-landing/index-landing.component.html [hunk 1]
```diff
@@ -23,9 +31,9 @@
     <img class="h-48 p-4 ring-1 rounded-lg mx-auto" src="/assets/images/onboarding-images/de4ZAy.tif.png" />
 </div>
 <div class="w-9/12 mx-auto mt-8 space-y-6 text-patchwrq">
-    <div class="ring-4 bg-patchwrq ring-gray-900 truncate shadow-md  text-center py-3 rounded-full">
+    <button class="w-full ring-4 bg-patchwrq ring-gray-900 truncate shadow-md  text-center py-3 rounded-full">
```
Comment (michael-gokey-architect) [inline src/app/index-landing/index-landing.component.html:34]: @theJayBecker can you make the buttons, into buttons?
the other two, so All three are

File: src/app/footer/footer.component.html [hunk 2]
```diff
@@ -1,19 +1,26 @@
...
           </div>
           <div class="pb-6">
-            <a href="#" class="text-sm leading-6 text-gray-600 hover:text-gray-900">Profile</a>
+            <a href="#" class="">
+              <img class="w-8 h-auto mx-auto" src="/assets/images/onboarding-images/biotool.png" />
+            </a>
           </div>
           <div class="pb-6">
-            <a href="#" class="text-sm leading-6 text-gray-600 hover:text-gray-900">Connecting</a>
+            <a href="#" class="">
```
Comment (michael-gokey-architect) [inline src/app/footer/footer.component.html:21]: @theJayBecker these look good, as they don't need to lead to anything yet. 
I think, we should put title text on them all, the footer, and in the body, so when you hover over them, the user sees a message
<a href="https://www.w3schools.com/html/" **_title="_**Go to W3Schools HTML section">Visit our HTML Tutorial</a>

File: src/app/footer/footer.component.html [hunk 3]
```diff
@@ -1,19 +1,26 @@
...
           <div class="pb-6">
-            <a href="#" class="text-sm leading-6 text-gray-600 hover:text-gray-900">Map</a>
+            <a href="#" class="">
+              <img class="w-10 h-auto mx-auto" src="/assets/images/onboarding-images/quest.png" />
+            </a>
           </div>
           <div class="pb-6">
-            <a href="#" class="text-sm leading-6 text-gray-600 hover:text-gray-900">Profile</a>
+            <a href="#" class="">
+              <img class="w-8 h-auto mx-auto" src="/assets/images/onboarding-images/biotool.png" />
```
Comment (michael-gokey-architect) [inline src/app/footer/footer.component.html:17]: @theJayBecker <img src="smiley.gif" **_alt="_**HTML tutorial" 
add alt texts, in future samples,

**Review by michael-gokey-architect — APPROVED** [review]
@theJayBecker @rorepopo All in all, it looks good. We need alt text and title text in all things moving forward. The more properly you make the HTML template, the less the angular developer has to do. They shouldn't have to be fixing the HTML, other than adding angular variables, and just writing the JavaScript, typescript code into the component class.
One of our stated requirements for website design is ADA compliance, which refers us to WACG 2+. So the more you can do to help out on the front end, the faster we can go, because your work is polished.  
I appreciate you looking at Robyne's designs and added the buttons instead of as <p> tags, and adding the drop shadow, and border color/shade/ thickness. 
It's all the little details that make us quality.

**Comment by theJayBecker** [comment]
Thanks for the feedback

added those tweaks to the page!

**Pull request merged**